master/buildbot/buildslave/libvirt.py

```diff
@@ -195,7 +186,7 @@ def _find_existing_instance(self):
         domains = yield self.connection.all()
         for d in domains:
             name = yield d.name()
-            if name.startswith(self.name):
+            if name.startswith(self.slavename):
```

Comment:
What do you mean? It was there before your patch.

Le jeu. 26 mars 2015 06:56, Mikhail Sobolev <notifications@github.com> a
écrit :

> In master/buildbot/buildslave/libvirt.py
> <https://github.com/buildbot/buildbot/pull/1609#discussion_r27190030>:
>
> > @@ -195,7 +186,7 @@ def _find_existing_instance(self):
> >          domains = yield self.connection.all()
> >          for d in domains:
> >              name = yield d.name()
> > -            if name.startswith(self.name):
> > +            if name.startswith(self.slavename):
>
> I'm inclined toward name. However, .name was not available. It probably
> needs to be fixed up in the chain...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1609/files#r27190030>.
>